### PR TITLE
Enable web profiles

### PR DESCRIPTION
**Pull request by brunobar79**
- Signing wallets and signatures are now generated on wallet creation.
- Web profiles are on by default
- For existing users - showcase is initialized once they share or add a new item to showcase
- For new users - it's enabled by default.
- Updated vercel url to main website

### CODE REVIEW

File: src/helpers/signingWallet.ts [hunk 1]
```diff
@@ -104,3 +82,36 @@ export async function signWithSigningWallet(
...
+    ? new Wallet(privateKey)
+    : await loadWallet(address, false);
+  if (mainWallet) {
+    const signatureForSigningWallet = await mainWallet.signMessage(
+      publicKeyForTheSigningWallet
+    );
+
+    const encryptor = new AesEncryptor();
+    const encryptedSignature = (await encryptor.encrypt(
+      RAINBOW_MASTER_KEY,
```
Comment (osdnk) [inline src/helpers/signingWallet.ts:103]: I think it's outside of this PR, but I share a strong concern about any gain from that bi-directional encryption.

File: src/model/migrations.js [hunk 2]
```diff
@@ -15,25 +14,20 @@ import {
...
 } from '@rainbow-me/handlers/localstorage/userLists';
-import networkTypes from '@rainbow-me/helpers/networkTypes';
+import { updateWebDataEnabled } from '@rainbow-me/redux/showcaseTokens';
 import { DefaultTokenLists } from '@rainbow-me/references';
 import logger from 'logger';
 
 export default async function runMigrations() {
   // get current version
   const currentVersion = Number(await getMigrationVersion());
+  //const currentVersion = Number(await getMigrationVersion()) - 1;
```
Comment (osdnk) [inline src/model/migrations.js:30]:
```suggestion
```

**Review by osdnk — APPROVED** [review]
go

**Review by skylarbarrera — APPROVED** [review]
🤾

**Pull request merged**